deployment/helm/skaha/sample-local-values.yaml

```diff
@@ -2,14 +2,19 @@
 
 # Skaha web service deployment
 deployment:
-  hostname: example.org # change this
+  hostname: example.org # Change this
   skaha:
+    image: images.opencadc.org/platform/skaha:0.16.1
```

Comment:
Please omit this as its already set in the main `values.yaml` file.

Reply:
Done